product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -0,0 +1,104 @@
+
+1. In the Replica Settings tab, Enter a name for your replica in the **Faraway Replica Name** field.
+
+1. Enter a password for your replica in the **Password** field.
+
+1. You cannot edit the **Database Type** field. The Postgres type and Postgres version are inherited from the parent cluster.  
+
+1. In the **Region** section, select the region where you want to deploy your replica.
+
+ You can select a region for deploying your replica even if it is not yet *Active*. Your replica creation request is added to a queue, and the replica is created after you activate the region. See [Activating regions](/biganimal/latest/getting_started/activating_regions) for more information.
+ 
+1.  The fields in the **Instance Type** section are prepopulated with values inherited from the parent cluster and are editable. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on instance type settings.
+
+1.  The fields in the **Storage** section are prepopulated with values inherited from the parent cluster. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the storage settings.
+ 
+1.  In the **Networking** section, specify whether to use private or public networking. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the Networking settings.
+
+1. Select **Create Replica**. 
+
+
+### Promote a faraway replica
+
+You can promote a faraway replica to a full-fledged cluster making it capable of accepting writes. The new cluster created is different from the parent cluster and it does not replace the parent cluster.
+
+1. Sign in to the [BigAnimal](https://portal.biganimal.com/) portal.
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the cluster with the replica you want to promote.
+
+1. In the **Overview** tab, you can see the cluster's replicas under **Faraway Replicas**.
+
+1. Select the *Promote replica* icon next to the replica you want to promote. The Create Cluster page appears. 
+
+1. The cluster settings are prepopulated with values inherited from the parent cluster. You can edit the cluster settings while creating your cluster. 
+
+   !!! NoteNotes
+     - A faraway replica can be promoted to a single node cluster or a high availablility cluster, but not an extreme high availability cluster. 
+     - **Cloud Provider**, **Postgres Type**, **Postgres Version**, and **Region** fields cannot be modified while promoting a replica to a cluster.
+   !!!
+
+1. Select **Promote Replica**. 
+ 
+
+### Modify a faraway replica
+
+To avoid lag during replication, do not reduce the capabilities of the replica by selecting settings lower than of the parent cluster.
```

Comment:
do not -> don't
I think you're missing words in the second part of the sentence:
by selecting settings that are lower than those of the parent cluster.
or ...that are lower than the parent cluster's.